warehouse/query-core/src/test/java/datawave/query/util/VisibilityWiseGuysIngest.java

```diff
@@ -0,0 +1,409 @@
+package datawave.query.util;
```

Comment:
Did we have to create a separate "Visibility" version of the WiseGuys ingest.  Can we not use this one for all WiseGuy tests?

Reply:
This class is used to test the aggregation of the visibilities while grouping. The original WiseGuys includes fabricated data models (for testing the expansion in functions). I'd rather test one thing at a time.